symbols/overlay29.yml

```diff
@@ -817,6 +889,14 @@ overlay29:
         r0: pointer to entity
         r1: speed counter weight
         return: speed stage
+    - name: CalcSpeedStageWrapper
+      address:
+        EU: 0x2300978
+        NA: 0x22FFF4C
+      description: |-
+        Calls CalcSpeedStage with r1 = 1.
```

Comment:
Since we know what r1 is, let's say "with a speed counter of 1"

Reply:
Just wanted to point out that r1 is the speed counter _weight_, to avoid confusion